1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,7 +21,9 @@ let basketballTeam = {
 */
 
 // write code here
-
+let basketballTopPlayerTeam=basketballTeam["topPlayers"]
+let sortTopAlfa=basketballTopPlayerTeam.sort()
```

Comment:
This is an interesting line of code. You should note that sort(), sorts the array in place, which means that basketballTopPlayerTeam is also sorted now, so you did not have to create a new variable called sortTopAlfa here.

Here, you are creating a reference to the first array basketballTopPlayerTeam, which means that if you change basketballTopPlayerTeam, there will also be an effect on sortTopAlfa, for example: see what happens if you run this code:

`let basketballTopPlayerTeam=basketballTeam["topPlayers"]`
`let sortTopAlfa=basketballTopPlayerTeam.sort()`
`basketballTopPlayerTeam.push("Luc Longley")`
`sortTopAlfa.forEach(x => console.log(x))`